### ISSUE TITLE
id based mount in fstab.json 

### ISSUE THREAD

**Issue by davidnuescheler** [adobe/helix-word2md#70]
Currently the fstab config consists a path to sharepoint eg.

```
{
  "mountpoints": [
    {
      "root": "/posts",
      "url": "https://adobe-my.sharepoint.com/personal/rofe_adobe_com/Documents/theblog/posts"
    }
  ]
}
```

...which leaks information about the user and the possibly parent folder structure. i think in many contexts this may not be a problem but it would be desirable in some cases to get to a place where we would use a more opaque id based identifier.

i could see a solution if there is an accessible id that's hidden somewhere in onedrive / sharepoint that we could use to identify a folder or alternatively some hash with a lookup. so a mountpoint url could look like this `https://adobe-my.sharepoint.com/?hash=<md5-of-path>`.
there may definitely be better ideas to cover for the leak of information...

**Comment by tripodsan** [comment]
yes, we can use the internal `drive-id`. but this is not visible via the normal drive UI in the browser.
we would need to create a tool to read it.
